tensorflow/compiler/xla/service/gpu/gpu_fusible.cc

```diff
@@ -131,5 +131,60 @@ bool ShapesCompatibleForMultiOutputFusion(const HloInstruction& instr1,
                                              get_loop_shape(instr_2));
 }
 
+bool IsInputFusibleScatter(const HloInstruction& instr) {
```

Comment:
You made this change in an earlier pull request. Please sync your branches to make the diff look right.